docs/customizations/exposing-prometheus-alertmanager-grafana-ingress.md

```diff
@@ -95,6 +97,7 @@ local kp =
     },
   };
 
+// Output a kubernetes List object with both ingresses (k8s-libsonnet)
```

Comment:
Added this note here as well, as the following caught me off-guard (as a complete newcommer to the jsonnet ecosystem).